File: HakerRank/data_structures/linked_lists/insert_a_node_at_the_head_of_a_linked_list.cs
/*
author: Daniel Lozano
source: HackerRank ( https://www.hackerrank.com )
problem name: Data Structures: Linked Lists: Insert a node at the head of a linked list
problem url: https://www.hackerrank.com/challenges/insert-a-node-at-the-head-of-a-linked-list/problem
*/

using System;

namespace HackerRank
{
    public class Program
    {
        public class Node
        {
            public int Data;
            public Node Next;
        }

        public static Node Insert(Node head, int x)
        {
            Node node = new Node() { Data = x };
            node.Next = head;
            return node;
        }

        public static void Main()
        {
           
        }
    }
}

File: HakerRank/data_structures/linked_lists/insert_a_node_at_the_head_of_a_linked_list.py
# author: Daniel Lozano
# source: HackerRank ( https://www.hackerrank.com )
# problem name: Data Structures: Linked Lists: Insert a node at the head of a linked list
# problem url: https://www.hackerrank.com/challenges/insert-a-node-at-the-head-of-a-linked-list/problem
# date: 9/1/2017

 class Node(object):
 
   def __init__(self, data=None, next_node=None):
       self.data = data
       self.next = next_node

def Insert(head, data):
    return Node(data, head)